config/foreman.migrations/20220406150414_foreman_google.rb

```diff
@@ -0,0 +1 @@
+answers['foreman::plugin::google'] = answers.delete('foreman::compute::gce')
```

Comment:
Looking closer at it, the problem is that this migration runs on `config/foreman-answers.yaml`. There is no answer for `foreman::compute::gce` so it's nil. That means it is essentially:
```ruby
answers['foreman::plugin::google'] = nil
```

I think a safe version would be:
```suggestion
old_answer = answers.delete('foreman::compute::gce')
answers['foreman::plugin::google'] ||= old_answer || false
```

The latter part (`|| false`) because `false || nil` is `nil`. The former part `||=` to respect if the answer is already `true`. The delete is on its own line because it must happen and if the answer is already true, it won't be evaluated.